.idea/GitAssemblyUtil.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Needs release notes

Reply:
all files from the .idea folder were unnecessary and deleted.

Reply:
? Was this a comment box mixup?

Reply:
It's an OS thing. https://rider-support.jetbrains.com/hc/en-us/articles/207097529-What-is-the-idea-folder-

Reply:
The comment you're replying to is

> Needs release notes

Reply:
The AssemblyUtil didn't have a RELEASE_NOTES.md when I forked it....

Reply:
Still needs release notes

Reply:
Release notes added.